src/components/projects-datas.js

```diff
@@ -0,0 +1,20 @@
+export const ProjectsDatas = [
```

Comment:
this is a variable not a component so the naming convention should be camel case for instance projectDatas

Reply:
okay, changed